SECURITY.md

```diff
@@ -0,0 +1,40 @@
+# Security policy for Flatpak
+
+ * [Supported Versions](#Supported-Versions)
+ * [Reporting a Vulnerability](#Reporting-a-Vulnerability)
+ * [Security Announcements](#Security-Announcements)
+ * [Security Vulnerability Response](#Security-Vulnerability-Response)
+
+## Supported Versions
+
+In stable branches and released packages, this table is likely to be outdated;
+please check
+[the latest version](https://github.com/flatpak/flatpak/blob/master/SECURITY.md).
+
+| Version  | Supported          | Status
+| -------- | ------------------ | -------------------------------------------------------------- |
+| 1.11.x   | :white_check_mark: | Development branch, releases may include non-security changes  |
+| 1.10.x   | :white_check_mark: | Stable branch, recommended for use in distributions            |
+| 1.9.x    | :x:                | Old development branch, no longer supported                    |
+| 1.8.x    | :white_check_mark: | Old stable branch, still supported                             |
+| <= 1.7.x | :x:                | Older branches, no longer supported                            |
+
+## Reporting a Vulnerability
+
+If you think you've identified a security issue in a Flatpak project,
+please DO NOT report the issue publicly via the Github issue tracker,
+mailing list, or IRC. Instead, send an email with as many details as 
+possible to [TODO: where?]
```

Comment:
Where is the right place? Personal mail to maintainers? A maintainer-only mailing list similar to `dbus-security`? The security contact address of a Flatpak-friendly distro like Red Hat or Endless?

Reply:
I think we should set up a maintainer-only flatpak-security list at freedesktop, next to the regular flatpak mailing list. Makes most sense I think. 

I also think maybe we should widen the set of maintainers a bit. @smcv, you're doing a lot of flatpak-related work recently. Would you be intereseted in helping out as a maintainer too?

Reply:
I've requested a list at https://gitlab.freedesktop.org/freedesktop/freedesktop/-/issues/340, please reply there to confirm.